standalone/runtime/src/lib.rs

```diff
@@ -1617,4 +1617,109 @@ impl_runtime_apis! {
             .and_then(|liquidity| liquidity.to_rpc_tuple())
         }
     }
+
+    impl zenlink_protocol_runtime_api::ZenlinkProtocolApi<Block, AccountId, CurrencyId> for Runtime {
```

Comment:
Why add these rpcs to standalone if they don't have an actual implementation?

Reply:
Because the `rpc` crate requires those runtime APIs and since we anyway plan to deprecate standalone I didn't want to add the actual Zenlink code impls there